pkg/controller/workload/job/job_controller.go

```diff
@@ -219,11 +244,30 @@ func (r *JobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 		return ctrl.Result{}, err
 	}
 
-	// 4.4 workload is admitted and job is running, nothing to do.
+	// workload is admitted and job is running, nothing to do.
 	log.V(3).Info("Job running with admitted workload, nothing to do")
 	return ctrl.Result{}, nil
 }
 
+func jobPodsReady(job *batchv1.Job) bool {
+	return job.Status.Succeeded+podsReadyCount(&job.Status) >= jobPodsReadyExpectedCount(&job.Spec)
+}
+
+func jobPodsReadyExpectedCount(jobSpec *batchv1.JobSpec) int32 {
```

Comment:
ditto, remove the job prefix

Reply:
Inlined the entire function as it wasn't called from other places